website/src/components/Step/index.css

```diff
@@ -7,12 +7,12 @@
   margin-top: 0;
 }
 
-.step h3 > i, html[data-theme='dark'] .step h2 > i {
+.step h3 > i, .step h2 > i {
   font-style: normal;
   color: #000;
 }
 
-html[data-theme='dark'] .step h3 > i, html[data-theme='dark'] .step h2 > i  {
+html[data-theme='dark'] .step h3 > i, html[data-theme='dark'] .step h2 > i {
```

Comment:
this isnt right. Will fix

Reply:
I think the actual issue is cache. It does work locally

Reply:
yup it was cache. Ran cloudfront invalidation, and now it's working
https://pr-1211.atmos-docs.ue2.dev.plat.cloudposse.org/quick-start/simple/write-components

![CleanShot 2025-04-22 at 15 27 01@2x](https://github.com/user-attachments/assets/3769f71c-dc05-48c9-8289-839d97c3625b)